spec/models/email_alert_spec.rb

```diff
@@ -128,13 +128,13 @@ def stub_create_content_change
       before do
         document.merge!(
           "links" => {
```

Comment:
I've replaced policies **in links** with other things. It doesn't seem to be possible to remove policies from tags. If my understanding is correct, email-alert-service is used by publishing api. The list of supported links/tags is [here](https://github.com/alphagov/email-alert-service/blob/c2bbc08a16af48cd4b5f4fd8dcc57ca533fdbee5/email_alert_service/models/major_change_message_processor.rb#L67). The only **tag** in this list which is also used by publishing api is `policies`.

Reply:
I think that, removing the tests for tags should go together with removing support for tags. Unless it's done this way, we will simply end up with untested, unused code.

Having said that, I don't think this is in the scope of the ticket behind this PR, which is about removing specialist topics. Removing tags seems like a separate, larger task and it would be better done by someone from the team responsible for this app who normally works on this code and understands it well (if there is such a person), instead of a person from another team. If there is no such person, I think it still should be another ticket. It is not really related to removing specialist topics.

With respect to using other kinds of tags, in another PR you explained that it would be good to make tests to reflect real data. Here the proposition is to go in the opposite direction - to introduce in the tests elements which cannot at this point reflect real data, if I understand things correctly. In some sense a deprecated field which was used in the past to me looks more like "real data" than a field which has never been used and can only be used potentially in the future.

From yet another perspective though, if there are confirmed plans to e.g. move the specialist publisher to publishing-api, I suppose it would make sense to use its data in tests as the preparation for the future.

What do you think?

Reply:
Eh, it's not really the opposite of the email-alert-api change - that was "can we make the data look more realistic for the tag", whereas this is "can we not use tags the don't exist". I mean, my ideal proposal here is more the last line: can we just get rid of the tests?

We're describing three options:
1) use tags that do exist (even if not currently valid)
2) get rid of the tags code altogether (and put it back if necessary later)
3) use tags that don't exist, 

If you don't want to do option 2, I think option 1 is clearly a better idea than 3

Reply:
OK, I will use the specialist publisher tags as a replacement for policy.

Reply:
@KludgeKML I've applied the changes.